### PR TITLE
Breadmaze

### PR DESCRIPTION
**Pull request by hamsterkoo**

### CODE REVIEW

**Review by LucasHT22 — CHANGES_REQUESTED** [review]
Thanks for PR! Good game! Could you add more levels?
And maybe add a mechanism to make it different from maze and skoban

**Comment by LucasHT22** [comment]
If you need help, you can call me! :)

**Comment by LucasHT22** [comment]
After adding the levels, I find it interesting to add a timer to make it more competitive:
`//timer time
var total = 10;
     var timer = setInterval(function(){
     total--;
     clearText();
     addText(""+total, { y: 1 , color: [255,0,0] });
       if(total <= 0){
         //for the player to not be able to complete the game after the timer ends
         clearTile(getFirst(player).x,getFirst(player).y);
         clearInterval(timer);
         addText("Game over!")
     }
     },1000);`

**Comment by hamsterkoo** [comment]
Hi,

Thanks for the feedback. 2 levels and a timer feature has been added.

**Review by LucasHT22 — COMMENTED** [review]
great changes! could increase the timer? oh, thanks for the credits :)

File: games/BreadMaze.js [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    setMap(currentLevel);
+    
+  }
+});
+
+
+
+function timerfunc() {
+
+  //timer - code borrowerd from flurffy
```
Comment (LucasHT22) [inline games/BreadMaze.js:232]: add ``var tempototal = `` and a time to not be able to count to -1

**Review by LucasHT22 — COMMENTED** [review]
Great changes! music is good :)

**Review by LucasHT22 — APPROVED** [review]
Congrats :) continue making games! you can get your sprig here! https://sprig-order.hackclub.dev/

**Comment by hamsterkoo** [comment]
Thanks for all the help!

**Pull request merged**